src/test/java/org/takes/facets/auth/social/PsLinkedinTest.java

```diff
@@ -54,113 +54,188 @@
  * @checkstyle MagicNumberCheck (500 lines)
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @checkstyle MultipleStringLiteralsCheck (500 lines)
- * @todo #686:30min After upgrade to qulice 0.16.4 we have to fix
- *  ExcessiveMethodLength PMD warings in this classe. Fix this warning
- *  and rid of the PMD suppression here.
  */
-@SuppressWarnings("PMD.ExcessiveMethodLength")
 public final class PsLinkedinTest {
 
     /**
+     * Field name for "First name".
+     */
+    private static final String FIRST_NAME = "firstName";
+
+    /**
+     * Test value for "First name".
+     */
+    private static final String FRODO = "Frodo";
+
+    /**
+     * Field name for "Last name".
+     */
+    private static final String LAST_NAME = "lastName";
+
+    /**
+     * Test value for "Last name".
+     */
+    private static final String BAGGINS = "Baggins";
+
+    /**
+     * Request path pattern for token endpoint.
+     */
+    private static final String TOKEN_PATTERN = "/uas/oauth2/accessToken";
```

Comment:
@mkordas it's a path that is meant to be used as a pattern :-) in this line
`new FkRegex(PsLinkedinTest.TOKEN_PATTERN, new TokenTake())`
[FkRegex(final String ptn, final String text)](https://github.com/yegor256/takes/blob/master/src/main/java/org/takes/facets/fork/FkRegex.java#L102)

It was already called `pattern` before my refactor https://github.com/yegor256/takes/blob/master/src/test/java/org/takes/facets/auth/social/PsLinkedinTest.java#L78